parlai/core/utils.py

```diff
@@ -713,6 +713,69 @@ def __contains__(self, key):
         """Determine if text contains any offensive words in the filter."""
         return self.contains_offensive_language(key)
 
+    def str_segment(self, text, dict_agent, max_length):
```

Comment:
should we add `k=1`... and change return type to list? (in case you want to try a few segmentations)